skimage/measure/_regionprops.py

```diff
@@ -648,4 +648,5 @@ def _install_properties_docs():
         setattr(_RegionProperties, p, property(getattr(_RegionProperties, p)))
 
 
-_install_properties_docs()
+if __debug__:
```

Comment:
Wth is that description https://docs.python.org/3.7/library/constants.html#__debug__:
```
This constant is true if Python was not started with an -O option.
```
Means that the default regime is a `debug` one? Does it make any sense to you? 🙂 

Regarding the change: it would be nice to have a short comment (with or without reference) on the necessity of the conditional.

Reply:
yeah... I didn't know about this debug/no debug more either. I guess default python is started in debug mode.

Apparently many software actually crashes if you try running it with `-OO`

Reply:
Effectively I think there are code still out there that crashes/misbehaves when executed with -O(O), which is a shame ;)